test/charTest.mo

```diff
@@ -0,0 +1,64 @@
+import Debug "mo:base/Debug";
+import Char "mo:base/Char";
+import Prim "mo:prim";
+
+/*
+//
+// Char.toUpper
+//
+
+assert(Char.toUpper('ö') == 'Ö');
```

Comment:
Does it work since it's `private`?

Reply:
It's commented out, but hang on, I'm making some doc changes.